common/js/src/random.js

```diff
@@ -36,12 +36,7 @@ const GSC = GoogleSmartCard;
  */
 GSC.Random.randomIntegerNumber = function() {
   const randomBytes = new Uint8Array(RANDOM_INTEGER_BYTE_COUNT);
-  if (typeof window !== 'undefined') {
-    window.crypto.getRandomValues(randomBytes);
-  } else {
-    // We're likely inside a Service Worker.
-    self.crypto.getRandomValues(randomBytes);
-  }
+  globalThis.crypto.getRandomValues(randomBytes);
```

Comment:
The change in this file is a pure refactoring: I didn't know about `globalThis` back then.